x-pack/plugins/triggers_actions_ui/public/types.ts

```diff
@@ -109,6 +109,8 @@ export interface AlertTypeModel {
   iconClass: string;
   validate: (alertParams: any) => ValidationResult;
   alertParamsExpression: React.FunctionComponent<any>;
+  isEditable: boolean;
+  consumer: string;
```

Comment:
it is make sense to rename it to 'uiConsumer' and I'm using server alert consumer as well to makes sure that this alert can has only its consumers alert types

Reply:
Maybe `producer`? Which is the one producing the alert type that a solution can then consume? Also +1 on it being server side, feature controls will need to know who is producing what alert type to enforce access controls.

Reply:
Agree on moving this to server side, if it can be helpful for a future embedding of the feature controls. I think `producer` is fine for a server side property name